src/seedu/addressbook/data/person/Address.java

```diff
@@ -56,4 +75,36 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+    public Block getBlockNum() {
+        return blockNum;
+    }
+
+    public void setBlockNum(Block blockNum) {
```

Comment:
Is there a need to use a setter in this case?
Do you need to set block number else where in the code base?
If there is no reason, there is no need to have the setter methods.